### PR TITLE
First Level Refactoring of vstest++ task

### PR DESCRIPTION
**Pull request by RanjanarMS**
First level refactoring for phases work

### CODE REVIEW

File: Tasks/VsTest/taskinputparser.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    tiaConfiguration.tiaRebaseLimit = tl.getInput('runAllTestsAfterXBuilds');
+    tiaConfiguration.fileLevel = tl.getVariable('tia.filelevel');
+    tiaConfiguration.sourcesDir = tl.getVariable('build.sourcesdirectory');
+    tiaConfiguration.tiaFilterPaths = tl.getVariable("TIA_IncludePathFilters");
+    tiaConfiguration.runIdFile = path.join(os.tmpdir(), uuid.v1() + '.txt');
+    tiaConfiguration.baseLineBuildIdFile = path.join(os.tmpdir(), uuid.v1() + '.txt');
+    tiaConfiguration.useNewCollector = false;
+    const useNewCollector = tl.getVariable('tia.useNewCollector');
+    if ( useNewCollector && useNewCollector.toUpperCase() === 'TRUE') {
+        tiaConfiguration.useNewCollector = true;
```
Comment (acesiddhu) [inline Tasks/VsTest/taskinputparser.ts]: hope by default this is false

File: Tasks/VsTest/models.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    buildPlatform: string;
+    testRunTitle: string;
+    vsTestVersion: string;
+    pathtoCustomTestAdapters: string;
+    tiaConfig: TiaConfiguration;
+}
+
+export interface DtaTestConfigurations extends TestConfigurations {
+    testConfigurationMapping: string; // TODO : What is this?
+    testSelection: string; // "testPlan" as selection string
```
Comment (acesiddhu) [inline Tasks/VsTest/models.ts]: tests?if you are planning to write tests. make sure we write in new way

**Comment by acesiddhu** [comment]
:shipit:

File: Tasks/VsTest/taskinputparser.ts (hunk 1 above)
Comment (RanjanarMS) [inline Tasks/VsTest/taskinputparser.ts]: I didn't change anything here.. just moved the code from vstest.ts

File: Tasks/VsTest/models.ts (hunk 2 above)
Comment (RanjanarMS) [inline Tasks/VsTest/models.ts]: I am planning last week for tests

**Pull request merged**